homeassistant/components/conversation/default_agent.py

```diff
@@ -419,9 +419,18 @@ def _get_or_load_intents(
                             encoding="utf-8"
                         ) as custom_sentences_file:
                             # Merge custom sentences
-                            merge_dict(
-                                intents_dict, yaml.safe_load(custom_sentences_file)
-                            )
+                            if isinstance(
+                                custom_sentences_yaml := yaml.safe_load(
+                                    custom_sentences_file
+                                ),
+                                dict,
+                            ):
+                                merge_dict(intents_dict, custom_sentences_yaml)
+                            else:
+                                _LOGGER.warning(
+                                    "Custom sentences file is empty path=%s",
```

Comment:
This message is also printed if it’s a list. We could probably just mention that “It’s not expected format”

Reply:
How about "Custom sentences file {path} needs to contain exactly one item in the expected format"?

Reply:
We don’t support lists at top level so it’s not the expected format. One item in the list wouldn’t work either